### ISSUE TITLE
Submerged trees

### ISSUE THREAD

**Issue by Frankess** [TerraForged/TerraForged#32]
Submerged trees are more realistic (as "roots") but at same this breaks a function of mod "Minecolonies". In there AI (Lumberjacks) look for trees to cut and it ignores anything that is submerged in ground (or are placed on cobblestone so trees used as decoration aren't cut down). 

Would be it possible to add an option in settings (or in config file) to not generate submerged trees?

**Comment by dags-** [comment]
Yep, I can add some sort of option for that.

In the meantime, you could try edit the json files in TF's mod jar under:
`data/terraforged/templates/trees`

They look something like this:
```json
{
  "type": "tree",
  "base": 3,
  "name": "terraforged:acacia_large",
  "paths": [
    "terraforged:trees/acacia/large"
  ]
}
```

You just need to modify the "base" value to zero. ie `"base": 0,` should do the trick

**Comment by dags-** [comment]
Fixed in 0.0.5
